xml/file_management.xml

```diff
@@ -148,49 +151,95 @@
 022</screen>
 
    <para>
-   The umask is subtracted from the access mode <literal>777</literal> if at
-   least one bit is set.
-   </para>
-
-   <para>
-    With the default umask, you see the behavior most users expect to see on a Linux system.
+    With the default umask, you see the behavior most users expect to see on a
+    Linux system.
    </para>
 
-<screen>
-&prompt.user;touch a
+<screen>&prompt.user;touch a
 &prompt.user;mkdir b
 &prompt.user;ls -on
 total 16
 -rw-r--r--. 1 17086    0 Nov 29 15:05 a
-drwxr-xr-x. 2 17086 4096 Nov 29 15:05 b
-</screen>
+drwxr-xr-x. 2 17086 4096 Nov 29 15:05 b</screen>
 
    <para>
     You can specify arbitrary umask values, depending on your needs.
    </para>
-<screen>
-&prompt.user;umask 111
+
+<screen>&prompt.user;umask 111
 &prompt.user;touch c
 &prompt.user;mkdir d
 &prompt.user;ls -on
 total 16
 -rw-rw-rw-. 1 17086    0 Nov 29 15:05 c
-drw-rw-rw-. 2 17086 4096 Nov 29 15:05 d
-</screen>
+drw-rw-rw-. 2 17086 4096 Nov 29 15:05 d</screen>
 
    <para>
-    Based on your thread model, you can use a stricter umask like <literal>037</literal>
-    to prevent accidental data leakage.
+    Based on your thread model, you can use a stricter umask like
```

Comment:
Should this be "threat model"?

Reply:
I would probably also put `umask` in [literal] tags.

Reply:
@cjschroder Thanks, fixed.

@lproven Even if umask is no English word, it's neither a command not a literal value in this context. Where it is one, I tagged it accordingly, but for other occurrences I suggest not tagging it. Just like we don't tag many other frequently used terms.

Reply:
> Where it is one, I tagged it accordingly, but for other occurrences I suggest not tagging it. Just like we don't tag many other frequently used terms.

OK, fair enough.

Ralph Waldo Emerson's line "A foolish consistency is the hobgoblin of little minds" probably applies. :-)